src/seedu/addressbook/Main.java

```diff
@@ -76,12 +78,28 @@ private void exit() {
         System.exit(0);
     }
 
-    /** Reads the user command and executes it, until the user issues the exit command.  */
+    /* Reads the user command and executes it, until the user issues the exit command.  
+     * Ensures that the file exists before executing command
+     * Warns user if filed disappears while program is running
+     */
     private void runCommandLoopUntilExitCommand() {
         Command command;
         do {
             String userCommandText = ui.getUserCommand();
             command = new Parser().parseCommand(userCommandText);
+            
+            try {
+            	String fileLoc = storage.getPath();
+            	FileInputStream checkFile = new FileInputStream(fileLoc); 
+            	checkFile.close();
+            	
+            } catch( FileNotFoundException fnfe ){
+            	System.out.println("Warning: File was deleted, recreating default storageFile");
+            } catch ( Exception e ){
+            	System.out.println(e);
+            }
```

Comment:
This method is probably too highlevel to do this low-level check. This method is simply executing commands. It doesn't even know there is a file involved.